LICENSE

```diff
@@ -0,0 +1,91 @@
+License text copyright (c) 2020 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+Parameters
+
+Licensor:             HashiCorp, Inc.
+Licensed Work:        The Licensed Work is (c) 2024 HashiCorp, Inc.
```

Comment:
Was it supposed to say `The Licensed Work is Copyright (c) 2024 HashiCorp, Inc.`

Reply:
I copied this from the Boundary repo, and it didn't have the `copyright` word. The bot however throws a validation error locally, and that's why I added yesterday but for now, I plan to have what Boundary and other products have just to rule out that this isn't causing the Github issue that we are seeing